Refactor EZO drivers to only use one sns ID

tichris0 opened the pull request:
Adds support for EZO humidity&temp sensor.

## Description:

Adding support for EZOHUM.  Without configuration, it will pick up the humidity only.  The user can enable the temperature through the sensor81 commands

## Checklist:
  - [X ] The pull request is done against the latest dev branch
  - [X ] Only relevant files were touched
  - [X ] Only one feature/fix was added per PR.
  - [X ] The code change is tested and works on Tasmota core ESP8266 V.2.7.4.5
  - [ ] The code change is tested and works on core ESP32 V.1.12.2
  - [X ] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

_NOTE: The code change must pass CI tests. **Your PR cannot be merged unless tests pass**_

tichris0 committed:
Add support for EZO HUM

Adds suppotr for EZO humidity&temp sensor.

arendst commented:
Would it be possible to consolidate all EZO sensors in one sns driver? This way I'm running out of sensor driver ids in no time while these sensors will hardly be used considering the price.

tichris0 commented:
> 
> 
> Would it be possible to consolidate all EZO sensors in one sns driver? This way I'm running out of sensor driver ids in no time while these sensors will hardly be used considering the price.

Yah it should be feasible.  I'll still keep the individual #define so that they can be enabled individually considering how little space Tasmota has on the esp8266.

tichris0 committed:
Refactor EZO drivers to only use one sns ID

Every EZO devices are now under xsns78.  They can still each be enabled through the appropriate USE_EZO* macro

tichris0 committed:
Merge branch 'development' of https://github.com/arendst/Tasmota into development

arendst commented:
Great! thx.
